src/parsing/parser.rs

```diff
@@ -2598,7 +2598,18 @@ fn parse_jsx_opening_element<'a>(node: &'a JSXOpeningElement, context: &mut Cont
         if context.config.jsx_attributes_prefer_single_line {
```

Comment:
The fix for most of the Deno issues is to turn this `jsx_attributes_prefer_single_line` from being `false` to `true` for deno's configuration. However, if we turn this on then we'll have to do a lot of validation to ensure it's working correctly. The hard part is basically going through and writing a lot of tests then fixing any issues.

That said, turning on `jsx_attributes_prefer_single_line` to `true` for deno is probably something we could do without causing many issues.

Reply:
To emphasize, fixing the deno issues is not as simple as just turning on more `prefer_single_line` configuration. I'd probably need to spend a lot of time redesigning things to fix all the issues. It would be very helpful to come up with tests though. You can add `(skip)` in a test's description to skip it.

Reply:
I see. Seems reasonable. I'll revert it.